frontend/global/global-styles.scss

```diff
@@ -35,31 +45,64 @@ button {
     background-color: $black;
     border: 0 none;
     color: $white;
-    font-size: 1.5em;
+    font-size: 1.4rem;
+    padding: 0.5rem 1rem;
     text-align: center;
     text-transform: uppercase;
-
-    &.small {
-        font-size: 1.1em;
-    }
+    outline: 0;
+    float: right; // For buttons outside of the button-split context
```

Comment:
I don't think we should float all buttons by default? It's currently messing up the "Join Team" page:

<table><tr><td>
Before:
<img src="https://user-images.githubusercontent.com/945577/34019307-0d262d60-e0e3-11e7-82e3-6a207a428a51.png">

</td><td>
After:
<img src="https://user-images.githubusercontent.com/945577/34019289-f81714e8-e0e2-11e7-8f17-c6347d1494b1.png">
</tr></table>

Reply:
Okay, I was just wanting anything that signified 'next' or 'confirm' to be on the right so that it's consistent throughout. But in practice it doesn't occur outside of the button-split very often so perhaps we could just use a special class.